doc/classes/TabContainer.xml

```diff
@@ -160,38 +165,39 @@
 			<argument index="0" name="visible" type="bool">
 			</argument>
 			<description>
-				If [code]true[/code] all the tabs will be visible.
+				If [code]false[/code], all tabs will be hidden completely (tab, content, and title). Defaults to [code]true[/code].
```

Comment:
- No comma after "If false"
- Style for defaults is "Default value: [code]true[/code]."

Reply:
fixed